package.json

```diff
@@ -16,7 +16,7 @@
     "autoprefixer": "^10.4.0",
     "axios": "^0.24.0",
     "html-metadata-parser": "^2.0.4",
-    "next": "^12.0.7",
+    "next": "^12.2.4",
```

Comment:
Any specific reason for updating Next.js?

Reply:
Not really. I didn't see any breaking changes that the package upgrade introduced. Would you prefer I roll it back to the previous version?

Reply:
Alright, no worries - just wanted to double check!